homeassistant/components/deconz/climate.py

```diff
@@ -24,6 +31,20 @@
     HVAC_MODE_OFF: "off",
 }
 
+DECONZ_PRESET_AUTO = "auto"
+DECONZ_PRESET_COMPLEX = "complex"
+DECONZ_PRESET_MANUAL = "manual"
+
+PRESET_MODES = {
+    DECONZ_PRESET_AUTO: "auto",
+    PRESET_AWAY: "holiday",
```

Comment:
So, preset modes are allowed to be custom.
Is there any specific reason why you made a translation?

Reply:
Umm, habit probably :D

I'll update it

Reply:
Fixed